envbuilder.go

```diff
@@ -724,10 +744,16 @@ func Run(ctx context.Context, options Options) error {
 	for _, env := range configFile.Config.Env {
 		pair := strings.SplitN(env, "=", 2)
 		os.Setenv(pair[0], pair[1])
+		exportEnv(pair[0], pair[1])
```

Comment:
Environment variables declared with `ENV` in the Dockerfile don't seem to actually show up in `configFile.Config.Env`. This is very strange and I'm not sure why this isn't working as expected.

Reply:
Weird, it works in the test but isn't working in my actual setup 🤔. Trying to dig into this...

Reply:
Ah, found the problem. Env vars from the Dockerfile get skipped when `SKIP_REBUILD=true` and we are starting an existing container. I will update this to skip writing the env export file in this scenario - we only want to write it when we do the actual build.